RELEASE.md

```diff
@@ -83,3 +88,33 @@ The usual steps are:
     * Copy the bundle files from `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle`
     * Copy `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle.Dockerfile` with the proper fix to the COPY commands
       (i.e. remove /bundle from the paths)
+
+--- 
+
+## Release file format.
+
+This example of the `release.toml` file uses tag `v1.0.1` as reference.
+
+```toml
+# FILE: ./release.toml
+[kuadrant]
+channel = "alpha"
+channels = ["alpha",]
+release = "1.0.1"
+
+[dependencies]
+Authorino_bundle = "0.16.0"
```

Comment:
I would call this `Authorino` instead of `Authorino_bundle`. `bundle` is OLM term, and release process covers OLM and helm charts.

Reply:
As long as we don't the bundle and component version going out of sync this also makes sense to me.